packages/react/src/auto/AutoTable.tsx

```diff
@@ -24,4 +24,6 @@ export type AutoTableProps<
   filter?: Options["filter"];
   actions?: TableOptions["actions"];
   excludeActions?: TableOptions["excludeActions"];
+  hideSearching?: boolean;
```

Comment:
I think this should just be `search: boolean` and `paginate: boolean`

Reply:
I'm not sure if people will think the `search` property can control the search query, not toggle the search bar. 🤔

Reply:
`searchable` would also work, but I think `search: boolean` is clear